packages/cli/src/scanner/page.ts

```diff
@@ -20,9 +20,9 @@ export class Page {
         @inject(WebDriver) private readonly webDriver: WebDriver,
     ) {}
 
-    public async create(): Promise<void> {
+    public async create(chromePath?: string): Promise<void> {
         if (isEmpty(this.browser)) {
-            this.browser = await this.webDriver.launch();
+            this.browser = await this.webDriver.launch(chromePath);
```

Comment:
I feel like chrome path should be a property of the web driver. Maybe we should pass it into the constructor of WebDriver?

Reply:
No, we only wanna expose it to the library, and we don't want any breaking changes in the CLI

Reply:
@haonliu, we may need to refactor this to may be look something like this - https://github.com/devaradhanm/hello-world-javascript-action/blob/master/browser/web-driver.ts

Reply:
but we can do that later